OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -42,6 +42,7 @@
     <ProjectReference Include="..\OpenRA.Game\OpenRA.Game.csproj">
       <Private>False</Private>
     </ProjectReference>
+    <PackageReference Include="DiscordRichPresence" Version="1.0.150" />
```

Comment:
We will need to fetch this from fetch-thirdparty-deps if we want to take this PR before #17744 (which IMO we do, ideally onto the prep branch).

OpenRA.nsi in the windows packaging scripts also needs to be updated to reference this dll.